examples/ext_authz/ext-authz-http-service.yaml

```diff
@@ -0,0 +1,76 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: ext-authz-http-service
+  name: ext-authz-http-service
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: ext-authz-http-service
+  template:
+    metadata:
+      labels:
+        app: ext-authz-http-service
+    spec:
+      containers:
+        - image: banaioltd/envoy-auth-basic-http-service:latest
```

Comment:
Maybe we should host this image in Kubeshop docker hub?

Reply:
Do you want to push the image? I don’t know if I have push rights to docker.io (<https://hub.docker.com/u/kubeshop>).

Suggested image name: `kubeshop/kusk-ext-authz-http-service:latest` and `kubeshop/kusk-ext-authz-http-service:v0.0.1`

Shall I also commit the `Dockerfile` for generating the `kusk-ext-authz-http-service` image.

Reply:
@mbana what's your DockerHub id? I can add you there so you can push...